drift/lib/src/runtime/api/connection_user.dart

```diff
@@ -471,7 +471,7 @@ abstract class DatabaseConnectionUser {
   /// See also:
   ///  - the docs on [transactions](https://drift.simonbinder.eu/docs/transactions/)
   Future<T> transaction<T>(Future<T> Function() action,
-      {bool requireNew = false}) async {
+      {bool requireNew = false, QueryInterceptor? interceptor}) async {
```

Comment:
We should also mention this in the documentation comment, e.g. something like `The optional [interceptor] can be used to apply a [QueryInterceptor] on the opened transaction. This is useful to modify the behavior of the executor for a specific transaction only.`